tendrl/gluster_integration/message/callback.py

```diff
@@ -9,14 +9,347 @@
+            severity="warning",
+            current_value="afr_subvol_down"
+        )
+        native_event.save()
+
+    def unknown_peer(self, event):
+        context = "unknown_peer" + event['message']['peer']
+        message = "Peer {} has moved to unknown state".format(
+            event['message']['peer']
+        )
+        native_event = NS.gluster.objects.NativeEvents(
+            context,
+            message=message,
+            severity="warning",
+            current_value="unknown_peer",
+            alert_notify=True
+        )
+        native_event.save()
+
+    def brickpath_resolve_failed(self, event):
+        context = "brickpath_resolve_failed" + event['message'][
+            "peer"] + event['message']["volume"] + event['message']["brick"]
+        message = "Brick path resolution failed for brick: {0} . Volume: {1}" \
+                  ".Peer: {}".format(
+                      event['message']["brick"],
+                      event['message']["volume"],
+                      event['message']['peer']
+                  )
+        native_event = NS.gluster.objects.NativeEvents(
+            context,
+            message=message,
+            severity="warning",
+            current_value="brick_path_resolve_failed",
+            alert_notify=True
+        )
+        native_event.save()
+
+    def quota_crossed_soft_limit(self, event):
+        context = "quota_crossed_soft_limit" + event[
+            "message"]["volume"] + event["message"]["path"]
+        message = "Quota soft limit crossed in volume: {0} for path: {1}" \
+                  ". Current usage: {2}".format(
+                      event['message']['volume'],
+                      event['message']['path'],
+                      event['message']['usage'],
+                  )
+        native_event = NS.gluster.objects.NativeEvents(
+            context,
+            message=message,
+            severity="warning",
+            current_value="quota_crossed_soft_limit",
+            alert_notify=True
+        )
+        native_event.save()
+
+    def bitrot_bad_file(self, event):
+        context = "bitrot_bad_file" + event['message'][
+            "brick"] + event['message']["path"] + event['message']["gfid"]
+        message = "File with gfid: {0} is corrupted due to bitrot." \
+                  "  Brick: {1}. Path: {2}".format(
+                      event['message']['gfid'],
+                      event['message']['brick'],
+                      event['message']['path'],
+                  )
+        native_event = NS.gluster.objects.NativeEvents(
+            context,
+            message=message,
+            severity="warning",
+            current_value="bitrot_bad_file",
+            alert_notify=True
+        )
+        native_event.save()
+
+    def afr_split_brain(self, event):
+        context = "afr_split_brain" + event['message']["subvol"]
+        message = "Subvolume: {} is affected by split-brain".format(
```

Comment:
What is meaning of affected by split brain? What is consequence of this? better we make this message more clear

Reply:
ack